### PR TITLE
Tk enable ecs pass role

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,7 +100,7 @@ The IAM policies are split into parts due to a limit on the size of the policies
 
 ### TDR AWS Accounts
 
-Three TDR Application AWS accounts are used to host the different environments:
+Two TDR Application AWS accounts are used to host the different environments:
 * Integration (intg)
 * Staging (staging)
 * Production (prod)
```
[the remark this answers is not in the document]
Comment (TomJKing) [reply README.md]: That's is what has been setup. We were given three AWS accounts: sandbox, integration and production. Laura said we were to have the staging and prod TDR environments on the production account. I believe this is the structure Digital Service have. This was discussed at the meeting with Digital Services when I wasn't here.

File: modules/permissions/specific-environment/permissions.tf [hunk 2]
```diff
@@ -0,0 +1,122 @@
+locals {
+  env_title_case = title(var.tdr_environment)
```
[the remark this answers is not in the document]
Comment (TomJKing) [reply modules/permissions/specific-environment/permissions.tf]: Sam found this

File: README.md [hunk 3]
```diff
@@ -2,17 +2,18 @@
...
+The purpose of this repository is to setup the necessary Terraform backend and Jenkins permissions AWS resources to support development operations for the TDR application in the different environments, using cross account access.
 
 Specific resources created:
 * AWS TDR Management account resources:
-  * **S3 Bucket**: contains the Terraform state files for each TDR workspace
-  * **DyanmoDb table**: used for locking to prevent concurrent operations on a single workspace
+  * **TDR state S3 Bucket**: contains the Terraform state files for each TDR workspace
+  * **TDR state DyanmoDb table**: used for locking to prevent concurrent operations on a single workspace
+  * **Jenkins state S3 Bucket**: contains the Terraform state files for Jenkins workspace
+  * **Jenkins state DyanmoDb table**: used for locking to prevent concurrent operations on a single workspace
```
[the remark this answers is not in the document]
Comment (TomJKing) [reply README.md]: Resolved

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TomJKing) [reply README.md]: Resolved

[the remark this answers is not in the document]
Comment (suzannehamilton) [reply README.md]: That's really clear now, thanks!

**Pull request merged**